packages/shared/src/components/onboarding/FilterOnboarding.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { cloudinary } from '../../lib/image';
+import OnboardingStep from './OnboardingStep';
+import FeedTopicCard from '../containers/FeedTopicCard';
+import { TagCategory } from '../../graphql/feedSettings';
+import { OnboardingFiltersLayout } from '../../lib/featureValues';
+
+interface FilterOnboardingProps {
+  tagsCategories: TagCategory[];
+  selectedId: Record<string, boolean>;
+  topicLayout?: OnboardingFiltersLayout;
+  onSelectedChange: (id: string) => void;
+}
+
+const classes: Record<OnboardingFiltersLayout, string> = {
+  grid: 'grid-cols-3 gap-6',
+  list: 'grid-cols-1 gap-4',
+};
```

Comment:
This can actually go in the common onboarding file, just to clean it up a bit.

Reply:
They are not used anywhere else outside this component, which is why I thought of keeping them here. Should we still store them in a common file?